tools/blast_to_scaffold/.shed.yml

```diff
@@ -0,0 +1,21 @@
+# .shed.yml supporting automatic pushes.
+owner: drosofff
+name: blast_to_scaffold
+description: Generate DNA scaffold from blastn or tblastx alignments of Contigs
+long_description: |
+  This tool start from DNA contigs that aligned to a subject DNA sequence through blastn or tblastx.
+  The contigs must be provided in fasta format. The blastn or tblastx output must be tabular, the 12 standard column plus column 13 with the length of the blastn or tblastx subject.
+  The sequence used to blastn or tblastx the contigs must be provided to serve as a guide to the final assembly
```

Comment:
`The sequence used to blastn or tblastx the contigs must be provided to serve as a guide to the final assembly` Maybe `The sequence used to BLAST (blastn or blastx) the contigs must be provided to serve as a guide for the final assembly` ?